jetty-http2/http2-common/src/main/java/org/eclipse/jetty/http2/HTTP2Connection.java

```diff
@@ -99,6 +101,26 @@ protected void setInputBuffer(ByteBuffer buffer)
             producer.setInputBuffer(buffer);
     }
 
+    public boolean isUseInputDirectByteBuffers()
```

Comment:
@gregw this is consistent everywhere, starting from `HttpConfiguration`.
I don't like "using" because it conveys to me that it may change, while instead I want to configure it in a way and remain in that way - therefore imperative "use".
"Direct" being adjective to "ByteBuffer" I put the two close together, which leaves "Input" and "Output" at the beginning of the name.

Alternative would be "useDirect[Input|Output]ByteBuffers?

Reply:
OK as is.